src/app/components/list-nugget-user/list-nugget-user.component.ts

```diff
@@ -37,7 +37,7 @@ export class ListNuggetUserComponent {
   getNuggets() {
     this.nuggetService.getListByUserId(this.itemsPerPage, (this.currentPage - 1) * this.itemsPerPage)
         .subscribe(result => {
```

Comment:
Il faut penser à se désabonner à chaque subscribe. Il y a plusieurs façons (https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f). Ca évite les fuites de mémoire